internal/integration/apprise/apprise.go

```diff
@@ -18,29 +20,53 @@ import (
 const defaultClientTimeout = 10 * time.Second
 
 type Client struct {
-	servicesURL string
-	baseURL     string
+	baseURL string
+	config  string
+}
+type AppriseConfig struct {
+	Default    string            `json:"default"`
+	Categories map[string]string `json:"categories"`
+	Feeds      map[string]string `json:"feeds"`
 }
 
-func NewClient(serviceURL, baseURL string) *Client {
-	return &Client{serviceURL, baseURL}
+func NewClient(baseURL string, config string) *Client {
+	return &Client{baseURL, config}
 }
 
 func (c *Client) SendNotification(entry *model.Entry) error {
-	if c.baseURL == "" || c.servicesURL == "" {
-		return fmt.Errorf("apprise: missing base URL or services URL")
+	if c.baseURL == "" {
+		return fmt.Errorf("apprise: missing base URL")
 	}
 
 	message := "[" + entry.Title + "]" + "(" + entry.URL + ")" + "\n\n"
 	apiEndpoint, err := urllib.JoinBaseURLAndPath(c.baseURL, "/notify")
 	if err != nil {
 		return fmt.Errorf(`apprise: invalid API endpoint: %v`, err)
 	}
+	var appriseConfig AppriseConfig
+	if err := json.Unmarshal([]byte(c.config), &appriseConfig); err != nil {
+		return fmt.Errorf("apprise: unable to deserialize config: %v", err)
+	}
+	var requestBody []byte
+	if _, exists := appriseConfig.Feeds[strconv.Itoa(int(entry.FeedID))]; exists {
+		requestBody, err = json.Marshal(map[string]any{
+			"urls": appriseConfig.Feeds[strconv.Itoa(int(entry.FeedID))],
+			"body": message,
+		})
+		// TODO - add function to fetch category related to this entry
```

Comment:
I do get null pointer on this one, need to check on how to mount categories with feed

Reply:
should we limit functionality to feed only?